returnn/extract_prior.py

```diff
@@ -1,12 +1,25 @@
-__all__ = ["ExtractPriorFromHDF5Job"]
+__all__ = [
+    "ExtractPriorFromHDF5Job",
+    "ReturnnComputePriorJob",
+    "ReturnnRasrComputePriorJob",
+]
 
+import copy
+import h5py
```

Comment:
h5py import should be together with other non-python-standard library imports. See https://github.com/rwth-i6/i6_core/blob/main/CONTRIBUTUNG.md (Import order). Also check other imports.